overlore/sqlite/vector_db.py

```diff
@@ -98,61 +112,39 @@ def get_townhalls_from_events(self, event_ids: list[int]) -> tuple[list[str], li
             - List of townhalls summary. One event_id of the list given in parameter must have been involved in the generation of the discussion.
             - Events_ids in the list given in parameter which haven't generated any summary before
         """
-        event_ids.extend([0 for i in range(0, 5 - len(event_ids))])
 
-        query = """
-            WITH GivenEventIDs(event_id) AS (VALUES (?), (?), (?), (?), (?)),
-            RecentDiscussions AS (
-                SELECT
-                    json_each.value AS event_id,
-                    T.summary,
-                    T.ts,
-                    T.rowid,
-                    ROW_NUMBER() OVER (PARTITION BY json_each.value ORDER BY T.ts DESC) as rn
-                FROM
-                    townhall T, json_each(T.events_ids)
-                WHERE
-                    json_each.value IN ((?), (?), (?), (?), (?)) AND json_each.value <> 0
-            ),
-            DupDiscussions AS (
-                SELECT
-                    event_id,
-                    summary,
-                    rowid,
-                    ROW_NUMBER() OVER (PARTITION BY rowid ORDER BY ts DESC) as dup_rn
-                FROM
-                    RecentDiscussions
-                WHERE
-                    rn = 1
-            )
-            SELECT
-                event_id,
-                summary
-            FROM
-                DupDiscussions
-            WHERE
-                dup_rn = 1
+        if len(event_ids) == 0:
+            return ([], [])
+
+        event_id_placeholders = (", ".join(["(?)"] * len(event_ids))).rstrip(",")
```

Comment:
`rstrip` is unnecessary, `join` only adds the separator in the middle of the members it joined

Reply:
so no separator will be a trailing character

Reply:
Yep, missed that one. Removed rstrip.